### DISCUSSION TITLE
How to add ip address fragments to existing IP pool range?

### DISCUSSION THREAD

**Discussion by nightzone** [HewlettPackard/POSH-HPOneView#559] [Q&A]
Under Addresses and Identifiers section we have
Subnet ID -> Address Ranges -> Address Fragments

For Subnet ID we can have few Address Ranges 
For Address Range we can have few address fragments

For example:
Subnet:
 Subnet ID 10.72.14.0
Address Ranges:
 Management               10.72.14.220 - 10.72.14.250
 10.72.14.0 Range 1      10.72.14.140 - 10.72.14.145, 10.72.14.148 - 10.72.14.150

With 
    New-OVAddressPoolSubnet -NetworkId 10.72.14.0 -SubnetMask 24 -Gateway 10.72.14.1
I can create Subnet 10.72.14.0
With 
   New-OVAddressPoolRange -IPSubnet $subnet -Name "Management" -Start 10.72.14.220 -End 10.72.14.250
   New-OVAddressPoolRange -IPSubnet $subnet -Name "10.72.14.0 Range 1" -Start 10.72.14.140 -End 10.72.14.145
I can create Ranges with one fragment in each

But how to add second fragment 10.72.14.148 - 10.72.14.150  to range "10.72.14.0 Range 1"   ?

I understand this is used not often, but possible.

**Comment by ChrisLynchHPE** [comment]
You create additional address ranges for the pool.  There is no way to reserve individual addresses within each range.  You would need to create sub-ranges within a subnet to get these "fragments".

**Reply by nightzone** [reply]
Hmmm... Via OV interface it is possible to add fragments inside of Range. So single range name, few IP fragments. I'll send screenshot by mail.
